docs/docs/topics/repository.mdx

```diff
@@ -117,3 +117,16 @@ Sync status keeps track of the synchronisation operation's output.
         Sync operation is ongoing.
     </TabItem>
 </Tabs>
+
+## Advanced actions
```

Comment:
I think we can choose another name than "advanced", perhaps just "Repository actions" or "Troubleshooting actions", though the import perhaps isn't troubleshooting. But to me "advanced" indicates that it's complicated in some way.

Reply:
Renamed!